### PR TITLE
Add internal elb for ingress

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/6913

Added only for legacy as ingress will be removed in clusterapi

### CODE REVIEW

**Comment by corest** [comment]
@giantswarm/sig-operator sry about pinging, but need to move here and this blocks me

**Comment by MarcelMue** [comment]
Why is this not added in the `cluster-api` branch? Is it not needed there?

**Comment by corest** [comment]
@MarcelMue from the first message

> Added only for legacy as ingress will be removed in clusterapi

File: service/controller/legacy/v30/adapter/guest_load_balancers.go [hunk 1]
```diff
@@ -111,6 +113,14 @@ func (a *GuestLoadBalancersAdapter) Adapt(cfg Config) error {
 	}
 	a.IngressElbScheme = externalELBScheme
 
+	// Ingress internal load balancer settings.
+	ingressInternalElbName, err := key.InternalLoadBalancerName(cfg.CustomObject.Spec.Cluster.Kubernetes.IngressController.Domain, cfg.CustomObject)
```
Comment (MarcelMue) [inline service/controller/legacy/v30/adapter/guest_load_balancers.go]: For some reason the names we generate here are too long for e.g. our E2E sometimes.
Not entirely certain why this happens or if it's just an E2E problem.
Screenshot from 2019-09-18 20-08-12

Comment (corest) [reply service/controller/legacy/v30/adapter/guest_load_balancers.go]: Crap
Maybe it needs to be shortened to IC
That’s anyway going to be removed with clusterapi

Comment (MarcelMue) [reply service/controller/legacy/v30/adapter/guest_load_balancers.go]: There is only one per CF stack, right? 
Can we just name it `ingress-<clusterID>-internal`?

Comment (corest) [reply service/controller/legacy/v30/adapter/guest_load_balancers.go]: That should be fine, yes

Comment (corest) [reply service/controller/legacy/v30/adapter/guest_load_balancers.go]: I don't get this. Is there some crazy stuff on CI only? LB name for internal ingress is `ingress-<cluster_id>-internal`, which is <32

Comment (MarcelMue) [reply service/controller/legacy/v30/adapter/guest_load_balancers.go]: Yes, I noticed this as well. Checking. I think the issue is that the clusterID in ci is way longer.

I will fix this, no need to worry for you.

**Pull request merged**